.github/workflows/nodejs.yml

```diff
@@ -13,7 +13,15 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: 12.x
-    - name: npm install, build, and test
+    - name: django-s3-file-field-client npm install, build, and test
+      working-directory: ./django-s3-file-field-client
```

Comment:
This directory name is a little awkward, imo. I don't think we have to change it now, since there's more cleanup to be done with the Django client and the examples. If you think of a better name, like `client-lib`, I'd be fine with it.